.travis.yml

```diff
@@ -4,4 +4,4 @@ node_js:
 before_script:
   - npm install grunt-cli -g
   - npm install
-script: grunt test
+script: npm run test
```

Comment:
@prasadtalasila It can be `npm test` or `npm run test`. As in the above code, he has put the testing as a script, he has used `npm run test`. Anyways following the best practices, it is better to put the script in `npm test` too. @utkarshrai take note.